account_invoice_rounding/account.py

```diff
@@ -113,11 +110,9 @@ def _compute_swedish_rounding(self, cr, uid, invoice, amounts,
         """
         Depending on the method defined, we add an invoice line or adapt the
         tax lines to have a rounded total amount on the invoice
-
         :param invoice: invoice browse record
         :param amounts: unrounded computed totals for the invoice
```

Comment:
It seems ```amounts``` is no more used.
Moreover, at https://github.com/hurrinico/account-invoicing/blob/porting_8_inv_round/account_invoice_rounding/account.py#L168 , ```amounts=[self.id]```